src/uct/api/v2/uct_v2.h

```diff
@@ -198,6 +209,49 @@ typedef struct uct_md_mem_dereg_params {
 } uct_md_mem_dereg_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_iface_is_reachable_v2.
+ */
+typedef struct uct_iface_is_reachable_params {
+    /**
+     * Mask of valid fields in this structure and operation flags, using
```

Comment:
we will not use field_mask to specify operation flags for this operation, please remove.
Would be something like `Mask of valid fields in this structure, using bits from...`

Reply:
fixed